special-interest-groups/sig-planner/member-list.md

```diff
@@ -0,0 +1,26 @@
+# SIG Planner Member List
+
+**NOTE**:
+
+* This member list is updated on a bi-weekly basis.
+* Last update time: 2019-11-24
+
+## Active Contributors
+
+Members and their contributions:
+
+None
```

Comment:
I think we may already have some, please check it carefully.

Reply:
@Reminiscent and @SeaRise is going to reach the requirements soon.
Currently is none.

Reply:
Update: they still need to waiting for current pr merged.